### ISSUE TITLE
PCB and Screen Sources

### ISSUE THREAD

**Issue by D3M05** [dannygrob/OpenWatch#1]
Hey Danny, great to see some more development on this and the addition of a touch screen.  Checking out your Altium files, it looks like a new pcb is needed with the new lines and a spot to connect the touchscreen.  Have you found a good source for getting these?  Also, I had a bit of trouble tracking down a good source for the screen beyond contacting the manufacturer directly.  Did you find anyone selling it directly?

Thanks! I look forward to helping with programming once I actually get this $1000 budget smartwatch working.  Oh, and please be sure to post up some detailed tutorials.  I'm going to need them!

**Comment by dannygrob** [comment]
Hi!

Yeah, you need a new pcb. Although my first version just had the connector wired up to the original smarchwatch pcb that was a hassle...

You can order it from https://jlcpcb.com (like I did), it cost me $17 for 10 pcbs and $16 shipping to the Netherlands. (I havn't even ordered it yet...)

You can use the original board parameters
https://github.com/S-March/smarchWatch_PUBLIC/blob/master/PCB%20Files/Watch/Board%20parameters.png

The LCD with touch is a bit hard. You can buy this one for testing (but as I said, the connector will not fold)
https://www.aliexpress.com/item/32920192414.html

I contacted the manufacturer directly on alibaba and payed $10 per LCD. It was not super difficult. If more people want to build this we can def. buy more at once and get a better price.
Here is the one I used:
https://www.alibaba.com/product-detail/TFT-LCD-Round-1-3inch-touch_62280453668.html?spm=a2700.galleryofferlist.0.0.16717253KS8UAr

**Comment by D3M05** [comment]
Awesome, wish I'd seen that jlcpcb site yesterday before I ordered a stencil.  Attempted my first solder by hand applying solder with a 22ga needle, and then heating it up with a hot air reflow gun.  Unfortunately it was a bit of a disaster.  connections on the screen connector had plenty of bridged pins, and I'm sure the situation under the dialog chip is probably the same.  I've got to wait for 0.39 ohm resistors anyway, so I'm hoping I'll have better luck on the second shot with a stencil.  Unfortunately the original BOM has a part number for a 390ohm resistor, so everything is on hold until then.

**Comment by D3M05** [comment]
Managed to get the pcb ordered from jlcpcb.  Says it's in production, but might be a while according to the banner saying they're affected by the coronavirus outbreak.  Ended up being about $50 total shipped including a stencil and 10pcbs.  Once you configure the settings it tends to add a bit.  Also waiting on hearing back from the screen manufacturer.  Hoping to get some clarification on the difference between TF13016C and the TF13016A-C.  Eventually I would like to start looking at 1.2 inch displays to try and start scaling down the size of the watch as much as possible.

**Comment by dannygrob** [comment]
Hi

Nice to hear you ordered the pcb with the stencil, let me know when you get them!

I did forgot something. I had ordered TF13016A-C and the backlight did not work. That's because the LED's are in series instead of in paralel on the other lcd. I had both so I could swap the backlight but that was not ideal. I asked Topfoison to make them paralel but that would mean a custom order and make it very expensive. I'd recommend buying [these](https://www.aliexpress.com/item/32920192414.html) at first.

Soldering without the stencil and solder paste is challenging! You need to be very capable :) Even with stencil you need to take your time. Good luck :)

ps. here is a short video of the menu and the screen in action
https://streamable.com/83sec

**Comment by D3M05** [comment]
Good to know about the Topfoison screen.  I'll do some hunting to see if I can find another better screen for the project.  I'll try the Surenoo for now.  Someone mentioned in the reviews that they were able to get it to bend, so maybe they fixed it or you got a bad batch.

I just printed your casing and charger dock, and I'm continually impressed by how much extra you've put into the project so far.  Love the updated design and sleekness.  I think you might have a wrong version uploaded though, because the top and bottom pieces of the watch housing do not match up on their screw tabs.  It's hard to tell without being able to join them, but did you have to increase the thickness of the watch to fit the touch screen, and are you still using the watch glass over the touch screen?

Very cool video!  I had no idea you had advanced that far on the UI.   Looking pretty legit!  Do you have it working for android?  I haven't even begun looking at that part of the project, but I can't imagine that will be too difficult.  iOS is usually 10x harder to get things working in from my experience.

**Comment by dannygrob** [comment]
If the old screen can bend it's connector I think that one is preferred! I had a case made for that one, I can always change the current one back!

The trick about my prints is that I use an SLA printer with resin to get super high res results (Longer orange 10). I did print the dock with a normal ultimaker FDM. I don't think a normal fdm printer can print this case correctly, as the screen slides in very tightly and it has a lot of overhangs that need to be precise. I had to make a lot of little adjustments. My bottom and top line up perfectly, I'll check if I got the right files in the repo, thanks!

It uses the apple bluetooth protocol ANCS right now, used from some dialog semi example by samson. The nice thing about this is that all you need to do is pair the watch and you'll receive messages right away, this works great. You don't need an app. For android you'd probably need one, but that could be made. I am a mobile dev by day :)

It's nice that someone else besides me is interested in this, also motivates me to finish it!
